packages/fluentui/react-icons-northstar/src/components/FocusModeIcon.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import cx from 'classnames';
+import createSvgIcon from '../utils/createSvgIcon';
+import { iconClassNames } from '../utils/iconClassNames';
+
+const FocusModeIcon = createSvgIcon({
+  svg: ({ classes }) => (
+    <svg role="presentation" focusable="false" viewBox="0 0 16 16" className={classes.svg}>
+      <g>
+        <path
+          className={cx(iconClassNames.outline, classes.outlinePart)}
```

Comment:
@AnastasiyaSarmant , please could you either remove the classnames at all (if the icon would not have distincting styles for `outline` and `filled`), like it's done in [CallDialpadIcon](https://github.com/microsoft/fluentui/blob/ee761dace1bb944941eb0ca3500f4cbee7f9b49c/packages/fluentui/react-icons-northstar/src/components/CallDialpadIcon.tsx#L14) for example. Or add a separate class and svg for `filled` icon like we have in [AcceptIcon](https://github.com/microsoft/fluentui/blob/ee761dace1bb944941eb0ca3500f4cbee7f9b49c/packages/fluentui/react-icons-northstar/src/components/AcceptIcon.tsx#L11)

Reply:
removed